spec/plans/prepare.fmf

```diff
@@ -59,12 +59,17 @@ example: |
         ``playbooks`` attribute.  Each of them will be applied
         using ``ansible-playbook`` in the given order. The path
         should be relative to the metadata tree root.
+
+        Use ``verbose`` attribute to enable different verbose levels
+        for ``ansible-playbook``. ``verbose`` will be applied only on
+        the debug level 3.
```

Comment:
Do I understand it correctly that it will be applied only if `tmt run -ddd` is used? Also I am wondering. Was the initial issue (#937 ) meant only for ansible or was it more general so that even other steps, e.g. provision could have verbosity enabled from a plan?

Reply:
It will be applied only if `tmt run -ddd` is used. But it's not a degradation, so I don't change anything there. It's unintuitive but I think it could be addressed in the separate issue. The same is about other steps. Maybe I'm wrong and it makes sense to fix all these cases in the same PR.

Reply:
I don't think it would be a good idea to allow overriding verbosity/debug level from plan config. This would mean that user could expect anything when running tests. It would be appropriate for user config (e.g. to set the default verbosity level) but I'd say it somehow does not fit into the plan.